01-messaging.md

```diff
@@ -316,7 +316,11 @@ For simplicity of diagnosis, it's often useful to tell a peer that something is
    * [`u16`:`len`]
    * [`len*byte`:`data`]
 
-The 2-byte `len` field indicates the number of bytes in the immediately following field.
+1. type: 1 (`warning`)
+2. data:
+   * [`channel_id`:`channel_id`]
```

Comment:
Clarification: This assumes / includes `temporary_channel_id` during funding of the channel?

Reply:
Correct